env.go

```diff
@@ -59,3 +60,18 @@ func loadConfigFile() (*Config, error) {
 	err = json.NewDecoder(f).Decode(&config)
 	return &config, err
 }
+
+func writeConfigFile(r io.Reader) error {
+	dir := filepath.Join(os.Getenv("HOME"), ".tranor")
+	err := os.MkdirAll(dir, 0755)
```

Comment:
I didn't wanted to handle the case where the directory already exists, but the side effect can be creepy, so I updated the code. Thanks!